core/iwasm/compilation/simd/simd_conversions.c

```diff
@@ -160,9 +160,9 @@ simd_integer_narrow_common(AOTCompContext *comp_ctx, AOTFuncContext *func_ctx,
 
     /* sat */
```

Comment:
would u mind quoting the spec restriction here , especially "regardless of the whether ..."